src/cli/tc.rs

```diff
@@ -55,31 +56,47 @@ pub fn run() -> Result<()> {
 }
 
 pub async fn init(file_path: &str) -> Result<()> {
+  let file_name = ".tailcallrc.graphql";
+  let file_name_yml = ".graphqlrc.yml";
   let tailcallrc: resource::Resource<str> = resource_str!("examples/.tailcallrc.graphql");
```

Comment:
About this resource_str expects a static string can't really substitute file_name dynamically